R/deprecated.R

```diff
@@ -633,3 +633,37 @@ dist_skel <- function(n, dist = FALSE, cum = TRUE, model,
   sample <- truncated_skel(n, dist = dist, cum = cum, max_value = max_value)
   return(sample)
 }
+
+#' Applies a threshold to all nonparametric distributions in a <dist_spec>
+#'
+#' @description `r lifecycle::badge("deprecated")`
+#' This function is deprecated. Use `bound_dist()` instead.
+#' @param x A `<dist_spec>`
+#' @param tolerance Numeric; the desired tolerance level. Any part of the
+#' cumulative distribution function beyond 1 minus this tolerance level is
+#' removed.
+#' @return A `<dist_spec>` where probability masses below the threshold level
+#' have been removed
+#' @keywords internal
```

Comment:
Do we need to deprecate internal code?

Reply:
This was an external function, it's the deprecation that has caused this to be converted to internal.